core/src/main/java/binnie/core/gui/fieldkit/WindowFieldKit.java

```diff
@@ -221,17 +221,11 @@ public void onWindowInventoryChanged() {
 				} else {
 					this.startAnalysing();
 					this.chromo.setSystem(null);
-					if (this.damageKit()) {
-						return;
-					}
 				}
 			} else if (!item.isEmpty()) {
 				this.isAnalysing = false;
 				this.analyseProgress = 1;
 				this.refreshSpecies();
-				if (this.damageKit()) {
```

Comment:
You should remove the `damageKit` method too, who knows how it was supposed to work.

Reply:
Ok, it is not used, just now noticed.